js/atoms/src/Info.svelte

```diff
@@ -9,11 +9,11 @@
 </div>
 
 <style>
-	div {
+	:global(.md) {
```

Comment:
:global is very dangerous and should be avoided when not scoped, this will affect *all* classes with md. You can scope it to be `div > :global(.md)` so it only targets the md inside div.

Reply:
Lemme take a look at the issue and see if there's a better solution